Source/monster.cpp

```diff
@@ -5008,6 +5013,9 @@ void MissToMonst(int i, Point position)
 	}
 }
 
+/**
+ * @brief Check that the given tile is avalible to the monster

```

Comment:
```suggestion
 * @brief Check that the given tile is available to the monster
```
(should probably fix the typo in the function name too 🤷 )